wordpress-mobile/WordPress-Android
# Detekt - Resolve/Suppress All Baseline Warnings - Long methods warnings

Parent: https://github.com/wordpress-mobile/WordPress-Android/issues/17010


This PR resolved/suppress all complexity related LongMethod warnings for the WordPress module (see docs [here](https://detekt.dev/docs/rules/complexity#longmethod)):

`15` x LongMethod (Resolve: https://github.com/wordpress-mobile/WordPress-Android/pull/17198/commits)

To test:

- There is nothing much to test here.

- Verifying that all the CI checks are successful should be enough (especially the detekt check).

- However, if you really want to be thorough, you could smoke test the WordPress and/or Jetpack apps to verify that everything works as expected on every screen that relates to these changes.


## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)
See `To test `section above.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
